packages/@react-native-windows/telemetry/src/utils/basePropUtils.ts

```diff
@@ -35,6 +38,29 @@ export async function deviceId(): Promise<string> {
   return '';
 }
 
+/**
+ * Gets the Windows build name, number and architecture.
+ * @returns A string containing the Windows build name, number and architecture.
+ * e.g. 19569.1000.amd64fre.rs_prerelease.200214-1419
+ */
+export async function fullBuildInfo(): Promise<string> {
+  try {
+    let regCommand = `${process.env.windir}\\System32\\reg.exe query ${DeviceIdBuildPath} /v ${DeviceIdBuildKey}`;
```

Comment:
Can you refactor the command part of this into a registry-key getting helper that can be shared with deviceId()?